packages/app-data/src/api/fetchDocFromAppData.ts

```
@@ -0,0 +1,53 @@
+ *
+ * This method will convert the app-data hex into a CID and fetch the content from IPFS.
+ *
+ * @param appDataHex App-data hex part of the order struct
+ * @param ipfsUri URL of the IPFS gateway to use for the fetch
+ *
+ * @returns a parsed AppData document
+ */
+export async function fetchDocFromAppDataHex(
+  appDataHex: string,
+  ipfsUri?: string
+): Promise<void | AnyAppDataDocVersion> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)